static/src/js/components/SpatialDisplay/__tests__/SpatialDisplay.test.js

```diff
@@ -205,406 +196,537 @@ describe('SpatialDisplay component', () => {
-      const input = '-77.119,38.791'
+    test('returns an error coordinates match each other.', async () => {
+      userEvent.setup()
+
+      const inputVal = '38.791,-77.119'
+
+      setup({
+        manuallyEntering: 'rectangle'
+      })
+
+      const swInput = screen.queryByTestId('spatial-display_southwest-point')
+      await userEvent.click(swInput)
+      await userEvent.type(swInput, inputVal)
+
+      const neInput = screen.queryByTestId('spatial-display_northeast-point')
+      await userEvent.click(neInput)
+      await userEvent.type(neInput, inputVal)
+
+      const updatedSwInput = screen.queryByTestId('spatial-display_southwest-point')
+      const updatedNeInput = screen.queryByTestId('spatial-display_northeast-point')
 
-      const boundingBoxResult = enzymeWrapper.instance()
-        .validateBoundingBoxCoordinates([input, input])
+      expect(updatedSwInput.value).toEqual(inputVal)
+      expect(updatedNeInput.value).toEqual(inputVal)
 
-      expect(boundingBoxResult).toEqual('SW and NE points contain matching coordinates. Please use point selection instead.')
+      const errors = screen.queryAllByTestId('filter-stack-item__error')
+      expect(errors).toHaveLength(1)
+
+      expect(errors[0].innerHTML).toEqual('SW and NE points contain matching coordinates. Please use point selection instead.')
     })
 
-    test('returns an error coordinates match each other and are invalid coordinates', () => {
-      const { enzymeWrapper } = setup()
-      const input = '-91.119,38.791'
+    test('returns an error coordinates match each other and are invalid coordinates', async () => {
+      userEvent.setup()
+
+      const inputVal = '-91.119,38.791'
+
+      setup({
+        manuallyEntering: 'rectangle'
+      })
+
+      const swInput = screen.queryByTestId('spatial-display_southwest-point')
+      await userEvent.click(swInput)
+      await userEvent.type(swInput, inputVal)
+
+      const neInput = screen.queryByTestId('spatial-display_northeast-point')
+      await userEvent.click(neInput)
+      await userEvent.type(neInput, inputVal)
+
+      const updatedSwInput = screen.queryByTestId('spatial-display_southwest-point')
+      const updatedNeInput = screen.queryByTestId('spatial-display_northeast-point')
+
+      expect(updatedSwInput.value).toEqual(inputVal)
+      expect(updatedNeInput.value).toEqual(inputVal)
 
-      const boundingBoxResult = enzymeWrapper.instance()
-        .validateBoundingBoxCoordinates([input, input])
+      const errors = screen.queryAllByTestId('filter-stack-item__error')
+      expect(errors).toHaveLength(1)
 
-      expect(boundingBoxResult).toEqual('Latitude (-91.119) must be between -90 and 90. SW and NE points contain matching coordinates. Please use point selection instead.')
+      expect(errors[0].innerHTML).toEqual('Latitude (-91.119) must be between -90 and 90. SW and NE points contain matching coordinates. Please use point selection instead.')
```

Comment:
same as above try deleting that data-test-id and then just finding what fails and replacing it with those assertions